pages/data-providers/[data-provider-id]/content.jsx

```diff
@@ -109,7 +111,7 @@ const Data = ({ store, ...restProps }) => {
       oai: v.identifier,
       title: v.title,
       author: v.author.map(a => a.name).join(' '),
-      'last-update': '12 days ago',
+      'last-update': dayjs(v.lastUpdate),
```

Comment:
It seems to be good. Better than Moment. Could you please check Temporal proposal for having something that would solve our problem? (for the future)

Reply:
I didn't know about Temporal proposal. Big disadvantage of it I can see is that it doesn't have human-readable formats. But it could be worth of try in the future.